src/garage/np/_functions.py

```diff
@@ -0,0 +1,74 @@
+"""Utility functions for NumPy-based Reinforcement learning algorithms."""
+import numpy as np
+
+from garage.misc import tensor_utils
+
+
+def process_samples_for_policy_gradient(paths_to_tensors, max_path_length,
+                                        baseline_predictions, discount):
+    """Return processed sample data based on the collected paths_to_tensors.
+
+    Args:
+        paths_to_tensors (list[dict]): A list of collected paths_to_tensors.
```

Comment:
sorry, my previous comment was unclear. this arg can be named `paths` (that's the convention around the codebase) and the function can be named `paths_to_tensors`